standards/WIT-SVID.md

```diff
@@ -0,0 +1,127 @@
+
+## Table of Contents
+
+## 1. Introduction
+
+The Workload Identity Token (WIT) is a token format specified by the IETF WIMSE working group in the [WIMSE Workload Credentials][1] document. The WIT binds a public key to the identity of a workload.
+
+The WIT-SVID is a sub-profiling of this token format for use in SPIFFE contexts. This document will not redefine key elements specified within the upstream [WIMSE Workload Credentials][1] document and as such the reader should be familiar with this document and its contents before implementing WIT-SVID.
+
+WIT-SVIDs are JSON Web Signature (JWS) data structures.
+
+### 2. JOSE Header
+
+This section describes the JOSE headers that may be included in the WIT-SVID.
+
+The WIT-SVID specification does not introduce any JOSE headers beyond those defined by the upstream document for the WIT. However, it does set additional restrictions and provide SPIFFE-specific guidance on some headers.
+
+TODO: Should we summarize `alg`/`typ` even if these will not differ from the WIMSE WIT?
+
+### 2.1. Key ID - `kid`
+
+The `kid` header is defined by the [JSON Web Signature (JWS)][2] document.
+
+For a WIT-SVID, this header is mandatory. This differs from the upstream WIT itself where this header is optional.
+
+The precise structure of this header is unspecified, and, it MUST be treated by verifiers as a case-sensitive string.
+
+The issuer MUST ensure that the value set within the `kid` header is unique to each issuing key-pair.
+
+### 2.2. Additional Headers
+
+It is permitted for an implementation to include additional headers not specified in this document or the upstream document.
+
+### 3. Claims
+
+This section describes the claims that may be included in the WIT-SVID.
+
+The WIT-SVID specification does not introduce any claims beyond those defined by the upstream document for the WIT. However, it does set additional restrictions and provide SPIFFE-specific guidance on some claims.
+
+TODO:
+
+- `iss` claim. This is RECOMMENDED in the WIT specification. Do we wish to mirror this, or, make it mandatory?
+
+### 3.1. Subject - `sub`
+
+The `sub` claim MUST be present and set to the SPIFFE ID of the workload to which it is issued. This is the primary claim against which workload identity is asserted.
+
+For example: `spiffe://example.org/service`.
+
+### 3.2. JWT ID - `jti`
+
+The `jti` claim provides a unique identifier for the WIT-SVID. It is defined by [RFC 7519][6].
+
+Primarily, this claim enables distinguishing one or more WIT-SVIDs that contain the same SPIFFE ID for the purposes of auditing. For example, if a validator of a WIT-SVID records the JTI within an audit log event, this audit log event can be correlated with the one emitted by the issuer, allowing the lineage of the credential to be ascertained.
+
+TODO: Do we repeat from RFC 7519 that the issuer MUST ensure that this field is unique?
+TODO: In WIT-SVID, is this a MAY/SHOULD/MUST?
+TODO: Do we make any comment here on revocation?
```

Comment:
That it could be used for this purpose due to its uniqueness property but that this is beyond SPIFFE specs.